cloud-config/google-cloud/README.md

```diff
@@ -0,0 +1,62 @@
+**To use this setup, you must be able create DNS domain entries. If you don't own a domain, services such as [FreeDNS](https://freedns.afraid.org) offer free sub-domains under a range of domains.**
+
+## Create the GCE VM
```

Comment:
I think it'd be nice if we give suggestions on location (close to where the data entry) and machine size.